repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/nhsd-list-article.ftl

```diff
@@ -28,20 +28,18 @@
 
 <div class="nhsd-t-grid nhsd-!t-margin-top-6">
     <div class="nhsd-t-row">
-        <div class="nhsd-t-col-3 nhsd-t-display-hide nhsd-t-display-l-show">
-            <div class="nhsd-a-box nhsd-a-box--border-grey nhsd-!t-margin-right-3 nhsd-t-display-sticky nhsd-t-display-sticky--offset-2">
+        <div class="nhsd-t-col-3 nhsd-!t-display-hide nhsd-!t-display-l-show">
+            <div class="nhsd-a-box nhsd-a-box--border-grey nhsd-!t-margin-right-3 nhsd-!t-display-sticky nhsd-!t-display-sticky--offset-2">
                 <h2 id="side-az-nav-heading" class="nhsd-t-heading-xs">Search A-Z</h2>
                 <@azList document "side-az-nav-heading"/>
             </div>
         </div>
         <div class="nhsd-t-col-l-9 nhsd-t-col-m-12">
-            <div class="nhsd-t-display-l-hide">
-                <h2 id="top-az-nav-heading" class="nhsd-t-heading-xs">Search A-Z</h2>
+            <div class="nhsd-!t-display-l-hide">
+                <h2 id="top-az-nav-heading" class="nhsd-!t-heading-xs">Search A-Z</h2>
```

Comment:
`nhsd-!t-heading-xs` isn't a valid override token. It was only the `nhsd-t-display-*` tokens that were updated.

Reply:
changed